documentation/polymer-templates/tutorial-template-mapped-components-limitations.asciidoc

```diff
@@ -0,0 +1,28 @@
+---
+title: PolymerTemplate, Mapped Components Limitations
+order: 15
+layout: page
+---
+
+=  PolymerTemplate, Mapped Components Limitations
+
+== Out-of-sync Methods
+
+In the Polymer template class example above, you could additionally map the `div` element with a `"footer"` identifier using the `Div` component and `@Id("footer")` annotation. However, neither the hierarchical structure, nor any attributes or properties, will be available on the server side using the Java API.
```

Comment:
This is now in a new section, to which example does "the Polymer template class example above" refer. Perhaps link to the example.

Reply:
Yes. Added examples for the Polymer template and its server-side representation.

Reply:
As I was reading this again, the text seem to imply that these limitations apply only to `PolymerTemplate`-based components. In fact they apply also to `LitTemplate`-based components. This should be mentioned even though the example is with polymer.

Reply:
@joheriks good point, added a not about it